lib/components/STableCell.vue

```diff
@@ -5,18 +5,19 @@ import STableCellAvatar from './STableCellAvatar.vue'
 import STableCellAvatars from './STableCellAvatars.vue'
 import STableCellDay from './STableCellDay.vue'
 import STableCellPill from './STableCellPill.vue'
+import STableCellPills from './STableCellPills.vue'
 import STableCellText from './STableCellText.vue'
 
 const props = defineProps<{
   name: string
   className?: string
-  cell?: TableCell | ((value: any, record: any) => TableCell)
+  cell?: any
   value: any
   record: any
   records: Record<string, any>
 }>()
 
-const cell = computed(() =>
+const computedCell = computed<TableCell | undefined>(() =>
```

Comment:
Had to rename this because Volar was getting confused in the template.

Reply:
Ah, gotcha. No problem 👍